### PR TITLE
BugFix: Correctly import classes and functions from arkane & submit.py

### PR DESCRIPTION
**Pull request by hwpang**
I modified the import in common.py and main.py so that it can import the classes and functions successfully from the newest arkane. I also modified a variable name in submit.py.

### CODE REVIEW

File: arc/common.py [hunk 1]
```diff
@@ -24,7 +24,12 @@
 import pandas as pd
 import qcelemental as qcel
 
-from arkane.ess import ess_factory, GaussianLog, MolproLog, OrcaLog, QChemLog, TeraChemLog
+from arkane.ess.factory import ess_factory
```
Comment (alongd) [inline arc/common.py:27]: Why was this change necessary? We should have these imports from the `__init__.py` in arkane.ess

Comment (hwpang) [reply arc/common.py:27]: Hi, I tried to run a test job with the original import lines, and met the following errors. These errors stop after I changed the import lines.

```
Traceback (most recent call last):
  File "/home/hwpang/Software/ARC/ARC.py", line 12, in <module>
    from arc.common import read_yaml_file
  File "/home/hwpang/Software/ARC/arc/__init__.py", line 5, in <module>
    import arc.main
  File "/home/hwpang/Software/ARC/arc/main.py", line 27, in <module>
    import arc.rmgdb as rmgdb
  File "/home/hwpang/Software/ARC/arc/rmgdb.py", line 16, in <module>
    from arc.common import get_logger
  File "/home/hwpang/Software/ARC/arc/common.py", line 27, in <module>
    from arkane.ess import ess_factory, GaussianLog, MolproLog, OrcaLog, QChemLog, TeraChemLog
ImportError: cannot import name 'ess_factory' from 'arkane.ess' (/home/hwpang/Software/RMG-Py/arkane/ess/__init__.py)
/var/spool/slurm/spool/job47983/slurm_script: line 43: deactivate: No such file or directory
```

Comment (kspieks) [reply arc/common.py:27]: Interesting. I wouldn't have expected that since I thought `__init__.py` took care of this

Comment (hwpang) [reply arc/common.py:27]: Correct me if I'm wrong,

If `ess_factory` is imported in `arkane/ess/__init__.py` by

`from arkane.ess.factory import ess_factory`

then `ess_factory` is only available at `factory` levle.

If you want to make `ess_factory` available in `ess` level,

I believe you should do 

`from ess.factory import ess_factory` in `arkane/ess/__init__.py`

Comment (alongd) [reply arc/common.py:27]: Very strange that you see this. Trying `from arkane.ess import ess_factory` on the current RMG master branch works fine for me, there's no reason for it to crush here. Could you verify which branches you're running on, and retry?

Comment (alongd) [reply arc/common.py:27]: I'm unfamiliar with this nuance. And it does seem to work for me. Again, very strange.

Comment (hwpang) [reply arc/common.py:27]: My RMG-Py is on the newest commit 3c397b719a97295fc9a9ed4f0791ce8e05678d08 on master branch. The weird thing is now it works even after I changed the import lines back. I will remove this commit then.

**Comment by alongd** [comment]
Please change the commit messages, the import change is not a bug (just adjasting to a recent change in RMG)

**Comment by hwpang** [comment]
> Please change the commit messages, the import change is not a bug (just adjasting to a recent change in RMG)

I changed the commit message for import.



> Looks good to me. Thanks for the PR!

Thanks!
